src/figuresClasses.ts

```diff
@@ -1,19 +1,109 @@
+// eslint-disable-next-line
+
 export interface Figure {
+  shape :string;
+  color: string;
+  a?: number;
+  b?: number;
+  c?: number;
+  radius?: number;
+  width?: number;
+  height?: number;
 
+  getArea: () => number;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  shape: string;
 
-}
+  color: string;
+
+  a: number;
+
+  b: number;
+
+  c: number;
 
-export class Circle {
+  constructor(
+    color: 'blue' | 'green' | 'red', a: number, b: number, c: number,
+  ) {
+    if (a <= 0 || b <= 0 || c <= 0) {
+      throw new Error('Length must be a positive number larger than 0.');
+    }
 
+    if (a + b <= c || a + c <= b || b + c <= a) {
+      throw new Error(
+        'Invalid triangle'
+        + 'any side must be smaller than the sum of the other two.',
+      );
+    }
+    this.shape = 'triangle';
+    this.color = color;
+    this.a = a;
+    this.b = b;
+    this.c = c;
+  }
+
+  getArea(): number {
+    const perimeter = (this.a + this.b + this.c) / 2;
+    const area = Math.sqrt(
+      perimeter
+      * (perimeter - this.a) * (perimeter - this.b) * (perimeter - this.c),
+    );
+
+    return Math.floor(area * 100) / 100;
```

Comment:
seems legit to introduce a rounding helper function to use it 3 times in your code